Makefile

```diff
@@ -189,8 +189,227 @@ bedrock-markdown-links:
 
 install-geth:
 	./ops/scripts/geth-version-checker.sh && \
-	 	(echo "Geth versions match, not installing geth..."; true) || \
- 		(echo "Versions do not match, installing geth!"; \
- 			go install -v github.com/ethereum/go-ethereum/cmd/geth@$(shell cat .gethrc); \
- 			echo "Installed geth!"; true)
+	(echo "Geth versions match, not installing geth..."; true) || \
+	(echo "Versions do not match, installing geth!"; \
+	go install -v github.com/ethereum/go-ethereum/cmd/geth@$(shell cat .gethrc); \
+	echo "Installed geth!"; true)
 .PHONY: install-geth
+
+### Rootstock ###
+
+rsk-regtest-log:
+	docker exec -ti -w "/var/lib/rsk/logs" rsk_regtest /bin/bash -c "tail -f rsk.log"
+.PHONY: rsk-regtest-log
+
+rsk-regtest-up:
+	docker start rsk_regtest && \
+	make rsk-regtest-log
+.PHONY: rsk-regtest-up
+
+rsk-regtest-down:
+	docker stop rsk_regtest
+.PHONY: rsk-regtest-up
+
+rsk-regtest-stop:
+	docker stop rsk_regtest
+.PHONY: rsk-regtest-stop
+
+rsk-regtest-delete:
+	docker rm rsk_regtest
+.PHONY: rsk-regtest-delete
+
+rsk-regtest-start:
+	docker run --name rsk_regtest -d -it \
+	-p 5050:5050 -p 4444:4444 -p 4445:4445 \
+  -v $(PWD)/rskj.logback.xml:/etc/rsk/logback.xml \
+  -v $(PWD)/rskj.node.conf:/etc/rsk/node.conf \
+  franciscotobar/rskj:optimism-6.1.0 --regtest --reset
+.PHONY: rsk-regtest-start
+
+take-5:
+	sleep 5
+.PHONY: take-5
+
+rsk-regtest-restart: rsk-regtest-stop	rsk-regtest-delete	rsk-regtest-start
+
+.PHONY: rsk-regtest-restart
+
+rsk-regtest-start-log: rsk-regtest-start take-5 rsk-regtest-log
+
+.PHONY: rsk-regtest-restart
+
+rsk-regtest-restart-log: rsk-regtest-restart take-5 rsk-regtest-log
+
+.PHONY: rsk-regtest-restart-log
+
+# https://docs.optimism.io/builders/chain-operators/tutorials/create-l2-rollup#build-the-optimism-monorepo
+rsk-build:
+	pnpm i &&  make op-node op-batcher op-proposer && pnpm build
```

Comment:
We may want to add the `--skipNxCache` to `pnpm build`, I had some issues of some libs not being built because of the cache and I fixed it with this flag. WDYT?

Reply:
done